verl/workers/actor/dp_actor.py

```diff
@@ -58,11 +58,11 @@ def __init__(self, config, actor_module: nn.Module, actor_optimizer: torch.optim
         self.actor_optimizer = actor_optimizer
 
         self.use_remove_padding = self.config.get("use_remove_padding", False)
-        if torch.distributed.get_rank() == 0:
```

Comment:
Can we skip this check based on a config instead of commenting it out. This would make it harder to merge back into the original verl repository